modules/nf-core/csvtk/join/main.nf

```diff
@@ -0,0 +1,55 @@
+process CSVTK_JOIN {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::csvtk=0.26.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/csvtk:0.26.0--h9ee0642_0':
+        'biocontainers/csvtk:0.26.0--h9ee0642_0' }"
+
+    input:
+    tuple val(meta), path(csv)
+    val in_format
+    val out_format
```

Comment:
These are not mandatory arguments of the tool, see further for suggestions on how to implement this in another way